docs/kyma/05-05-profiles.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Profiles
+type: Configuration
+---
+
+The Kyma Operator allows you to use pre-defined profiles to install or upgrade Kyma. To install Kyma using a profile, you must specify the **spec.profile** attribute in the [Installation custom resource](#custom-resource-installation) (CR).
+
+The profile is a subset of the default values.yaml defined in `profile-{name}.yaml` file at the component chart folder and can override only a section, or even the whole file if required. If the profile file is not present, the Kyma Operator will use default set of values from the `values.yaml` file.
+Currently supported profiles are: 
+- evaluation
+- production
+
+The following fragment of the `values.yaml` shows definition limitRange settings:
+
+```yaml
+limitRange:
+  max:
+    memory: 4Gi
+  default:
+    memory: 96Mi
+  defaultRequest:
+    memory: 32Mi
+```
+
+To override `limitRange.max.memory` put the following in the `profile-{name}.yaml` file: 
+
+```yaml
+limitRange:
+  max:
+    memory: 5Gi
+```
+
+>**NOTE:** If the `profile-{name}.yaml` file doesn't exist in a component chart you can simply create it.
+
+On top of the applied profile the Kyma Operator will put user-defined overrides. For more information, see the [overrides section](#configuration-helm-overrides-for-kyma-installation).
```

Comment:
```suggestion
>**NOTE:** User-provided overrides have precedence over the applied profile. For more information, see the [overrides section](#configuration-helm-overrides-for-kyma-installation).
```

Reply:
2 notes one after another? wouldn't it look a bit strange?

Reply:
@klaudiagrz what do you say?

Reply:
We try to avoid such situations but we also already have such cases in the documentation so I wouldn't worry about that too much 🙂

Reply:
OK, it doesn't have to be a NOTE. I'll remove the emphasis to avoid double-notes.